scripts/en2ja.json

```diff
@@ -0,0 +1,104 @@
+{
+  "title": "English-to-Japanese string replacement",
+  "description": "Replace EN string 'from' with associated JA string 'to'.",
+  "pairs": [
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/asa/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/asa/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/eu/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/eu/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oag/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oag/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/oie/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/oie/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "rel=\"canonical\" href=\"https://help.okta.com/wf/en-us",
+      "to": "rel=\"canonical\" href=\"https://help.okta.com/wf/ja-jp",
+      "note": "Many (but not all) topics contain a canonical link in the header."
+    },
+    {
+      "from": "\"https://help.okta.com/en-us/Content/index.htm\"",
+      "to": "\"https://help.okta.com/ja-jp/Content/index.htm\"",
+      "note": "This link is in the top-level menu bar in all topics."
+    },
+    {
+      "from": "\"https://help.okta.com/en-us/Content/Topics/ReleaseNotes/okta-relnotes.htm\"",
+      "to": "\"https://help.okta.com/ja-jp/Content/Topics/ReleaseNotes/okta-relnotes.htm\"",
+      "note": "This link is in the top-level menu bar in all topics."
+    },
+    {
+      "from": "\"https://support.okta.com/help/s/\"",
+      "to": "\"https://support.okta.com/help/s/?language=ja\"",
+      "note": "Apply query param to direct users to their language."
+    },
+    {
+      "from": "?language=en_US",
+      "to": "?language=ja",
```

Comment:
Okta Support is slowly rolling out lang support to the various assets. I had noticed that one site that had previously not had JA support now provides it. With that in mind, I added lang query params -- if they add lang support, then we redirect to that automatically. If not, the site simply renders in EN. Otherwise, we'd have to keep track of when Support adds JA to each asset. 
At any rate, that was my reasoning here.